<launch>
    <node pkg="model_prediction" type="generator.py" name="generator" output="screen" />

    <node pkg="model_prediction" type="model_prediction.py" name="predictor" output="screen" >
        <param name="trigger" value="command"/>
        <param name="command_type" value="geometry_msgs/Twist"/>
        <param name="command_field" value="linear.x"/>
        <param name="command_coef_csv" value="-0.03343,0.02551,0.1098,0.03073"/>
        <param name="state_type" value="geometry_msgs/TwistStamped"/>
        <param name="state_field" value="twist.linear.x"/>
        <param name="state_coef_csv" value="-0.0143,-0.08758,-0.2148,-0.5688"/>

        <remap from="/predictor/command" to="vrep/twistCommand"/>
        <remap from="/predictor/state" to="vrep/twistStatus"/>
    </node>

</launch>

